spec/integration/migration_hash_and_index_spec.rb

```diff
@@ -137,8 +137,8 @@
     create_range_partition_of_subpartitioned_by_list
 
     adapter.add_index_on_all_partitions table_name, :updated_at, name: index_prefix, using: :hash,
-                                        algorithm: :concurrently,
-                                        where: "created_at > '#{current_date.to_time.iso8601}'"
+                                                                 algorithm: :concurrently,
+                                                                 where: "created_at > '#{current_date.to_time.iso8601}'"
```

Comment:
This looks pretty nasty

Reply:
Updated to match style, described above.